src/OpenRiaServices.Hosting.Wcf/Framework/WCF/Behaviors/WebHttpQueryStringConverter.cs

```diff
@@ -17,13 +17,20 @@ namespace OpenRiaServices.Client.Web.Behaviors
 {
     internal class WebHttpQueryStringConverter : QueryStringConverter
     {
+        const string DateTimeFormat = @"yyyy\-MM\-ddTHH\:mm\:ss.FFFFFFFK";
+        private static readonly DataContractJsonSerializerSettings s_jsonSettings
+            = new DataContractJsonSerializerSettings()
+            {
+                UseSimpleDictionaryFormat = true,
```

Comment:
it seem the serializer is a bit buggy, keys of dictionaries are not formatted according to datettimeformat so we cannot improve the ditionary format this now